src/addBusinessDays/index.ts

```diff
@@ -37,110 +36,63 @@ export default function addBusinessDays(
   requiredArgs(2, arguments)
   const options = dirtyOptions || {}
   const exceptions = options.exceptions || {}
-  const businessDays =
-    options.businessDays == null
-      ? [1, 2, 3, 4, 5]
-      : options.businessDays.filter((number) => number < 7).map(toInteger)
+  const businessDays = options.businessDays ?? [1, 2, 3, 4, 5]
 
-  const date = toDate(dirtyDate)
-  const amount = toInteger(dirtyAmount)
-  let accountedForExceptions = 0
-  const isExcepted = (date: Date): boolean | null => {
-    if (options.exceptions) {
-      const exception =
-        Object.keys(options.exceptions).find((e) =>
-          isSameDay(new Date(e), date)
-        ) || ''
-      if (options.exceptions[exception] !== undefined) {
-        accountedForExceptions += 1
-      }
-      return options.exceptions[exception]
-    }
-    return null
+  // Throw an exception if businessDays includes a number greater than 6
+  if (businessDays?.filter((number) => number > 6).length > 0) {
+    throw new RangeError('business days must be between 0 and 6')
```

Comment:
thought: I was going to point out that the use of an array here seems like a poor choice, since one could specify `[6, 6, 6]`... however, I noticed that the new implementation at least is no longer using `businessDays.length` in its calculations, and wouldn't be susceptible to duplicate number issues. So won't make a blocking request for change, but might be worth circling back to that in the associated tech debt ticket.

Reply:
I added a test to check for this, so if we refactor later and add the `businessDays.length` back in, that test will fail.

Reply:
Great, TY 👍